Source/ACE/Entity/Landblock.cs

```diff
@@ -483,7 +493,9 @@ public void UseTime()
                     QueuedGameAction examination = player.ExaminationQueuePop();
                     if (examination != null)
                         HandleGameAction(examination, player);
+                    curplayer++;
                 });
+                UpdateStatus(curplayer);

```

Comment:
This is a really crude way to collect a player count.  You could have done "UpdateStatus(allplayers.Count);" instead.